api-reference/10 UI Widgets/dxFileManager/5 File System Providers/Object/Object.md

```diff
@@ -40,6 +40,8 @@ The following code shows how to bind the **FileManager** to the **Object** file
 
 ---
 
+[note] [Register the JSZip library](/concepts/40%20Angular%20Components/10%20Getting%20Started/03%20Add%20DevExtreme%20to%20an%20Angular%20CLI%20Application/08%20Register%203rd-Party%20Dependencies%20in%20Angular%20CLI%206+/01%20JSZip%20Registration.md 'Documentation/Guide/Angular_Components/Getting_Started/Add_DevExtreme_to_an_Angular_CLI_Application/#Register_3rd-Party_Dependencies_in_Angular_CLI_6+/JSZip_Registration') to download several files at once.
```

Comment:
The link you're giving is for Angular only. See what we did in a similar situation for the client-side export:
https://js.devexpress.com/Documentation/20_1/Guide/Widgets/DataGrid/Client-Side_Exporting/#Prerequisites_and_Restrictions

Reply:
Removed a note